datafusion/core/src/datasource/file_format/parquet.rs

```diff
@@ -1257,13 +1265,13 @@ mod tests {
     }
 
     async fn get_exec(
-        ctx: &SessionContext,
+        ctx: &SessionState,
```

Comment:
I find the `ctx` name confusing

Reply:
Yeah, I was just copying the naming used in things like TableProvider - I don't disagree that it is confusing. Renamed